src/test/scala/de/zalando/beard/renderer/BeardTemplateRendererSpec.scala

```diff
@@ -195,6 +195,16 @@ class BeardTemplateRendererSpec extends FunSpec with Matchers {
       }
     }
 
+    describe("render a Comment Statement") {
```

Comment:
It would be nice to have a test for the parser too. In my opinion, a template with only comments should produce an empty AST.